iwyu_output.h

```diff
@@ -108,8 +108,8 @@ class OneUse {
   string comment_;                 // If not empty, append to clang warning msg
   vector<string> public_headers_;  // header to #include if dfn hdr is private
   string suggested_header_;        // header that allows us to satisfy use
-  bool ignore_use_;                // set to true if use is discarded
-  bool is_iwyu_violation_;         // set to false when we figure out it's not
+  bool ignore_use_ = false;        // set to true if use is discarded
```

Comment:
Ah, I see what happened now. Can we please keep initialization in the init-list? Better to have initialization in one place.